README.md

```diff
@@ -152,14 +152,47 @@ public class Startup
 {
   public void ConfigureServices(IServiceCollection services)
   {
-      services.AddFeatureManagement();
+      services.AddFeatureManagement()
+              .AddFeatureFilter<MyCriteriaFilter>();
   }
 }
 ```
 
-This tells the feature manager to use the "FeatureManagement" section from the configuration for feature flag settings.
+This tells the feature manager to use the "FeatureManagement" section from the configuration for feature flag settings. It also registers a customized feature filter `MyCriteriaFilter`. Please refer to the `Implementing a Feature Filter` section below for more details.
 
-**Advanced:** The feature manager looks for feature definitions in a configuration section named "FeatureManagement". If the "FeatureManagement" section does not exist, the configuration will be considered empty.
+You can also specify that feature management configuration should be retrieved from a different configuration section by calling configuration.GetSection and passing in the name of the desired section. The following example tells the feature manager to read from a different section called "MyFeatureFlags" instead:
+``` C#
+…
+services.AddFeatureManagement(configuration.GetSection("MyFeatureFlags"));
+…
+```
+
+**Advanced:** By default, the feature manager retrieves feature flag configuration from the "FeatureManagement" section of the .NET Core configuration data. If the "FeatureManagement" section does not exist, the configuration will be considered empty. When a specific configuration section is provided to `AddFeatureManagement()`, the feature manager will fall back to the root of that configuration section to look for feature definitions, if there is no "FeatureManagement" section found. 

```

Comment:
> When a specific configuration section is provided to `AddFeatureManagement()`, the feature manager will fall back to the root of that configuration section to look for feature definitions, if there is no "FeatureManagement" section found.

I think we can simplify this to

"When a specific configuration section is passed when registering feature management, the "FeatureManagement" section is not needed."

Reply:
Actually, I might suggest that we move this "Advanced" note to be directly under the section detailing the plain "AddFeatureManagement", and then move on to describing the alternative approach which lets you pass a configuration section directly. In the alternative approach I think it should be clear enough by our explanation that the section is where the flags are defined.